views/layout.pug

```diff
@@ -77,19 +77,25 @@ html(lang='en', prefix='og: http://ogp.me/ns#', itemscope, itemtype='http://sche
                         .col-sm-12.d-none.d-lg-block
                             .twitter-timeline-custom
 
-        script(src=config.javascript.jquery.uri,
-            integrity=config.javascript.jquery.sri, crossorigin='anonymous')
-
         -var bootstrap = config.bootstrap4.filter(function(o) { return o.latest; })[0];
-        script(src=bootstrap.javascriptBundle, integrity=bootstrap.javascriptBundleSri,
-            crossorigin='anonymous')
 
         script(nonce=nonce)
-            include /assets/js/clipboard.min.js
+            include /assets/js/vendor/clipboard.min.js
+            include /assets/js/vendor/loadjs.min.js
 
             if (process.env.NODE_ENV === 'production')
                 include:uglify-js /assets/js/main.js
             else
                 include /assets/js/main.js
 
+            | loadjs(['#{config.javascript.jquery.uri}', '#{bootstrap.javascriptBundle}'], 'jquery', {
```

Comment:
If you can't move all of this in to the include, per your comment, I would just move the contents of the include in to the pug template as well. It will read better.

Reply:
I don't think that's possible because of the variables. If you make it work, let me know.

Reply:
I mean, move it to a separate JS file. A pug file should work fine, though.